docker-compose/Makefile

```diff
@@ -81,30 +83,58 @@ download-cli:
 
 
 .PHONY: run
-run: print-host check-required-ports setup start-docker-compose init-couchdb init-whisk-cli
+run: print-host check-required-ports setup start-docker-compose init-couchdb quick-start-pause init-whisk-cli init-route-mgmt init-whisk-properties
 
 print-host:
 	echo "host ip address: ${DOCKER_HOST_IP}"
 
+.PHONY: init-whisk-properties
+init-whisk-properties:
```

Comment:
it's interesting to note that copying `whisk.properties` in the `PROJECT_HOME` folder AND executing ```docker run -d -p 3128:3128 style95/squid:3.5.26-p1```, developers can also execute the tests ( i.e. `./gradlew testLean` ). Something to keep in mind for an upcoming PR.  @csantanapr has nicely captured the setup as part of https://github.com/apache/incubator-openwhisk/blob/master/ansible/README.md#docker-for-mac-users